jobs/transforms/user_validation_test.py

```diff
@@ -160,6 +160,41 @@ def test_model_not_marked_as_deleted_recently(self):
         self.assert_pcoll_equal(output, [])
 
 
+class UserEmailPreferencesModelRelationshipsTests(
+        job_test_utils.PipelinedTestBase):
+
+    NOW = datetime.datetime.utcnow()
+
+    def test_model_without_dependent_model(self):
+        model = user_models.UserEmailPreferencesModel(
+            created_on=self.NOW, id='test_user',
+            last_updated=self.NOW)
+        output = (
+            self.pipeline
+            | beam.Create([model])
+            | beam.ParDo(
+                user_validation.user_email_preferences_model_relationships(
+                    model))
+        )
+        self.assertIsNot(output, [])
```

Comment:
what is the expected output? Also in which exactly it will fail and what should be the check for that?

Reply:
@rohitkatlaa

Reply:
So if there is no UserSettingModel with id as test_user then there should be an error